airbyte-integrations/connectors/destination-postgres/src/main/resources/spec.json

```diff
@@ -56,16 +56,165 @@
       },
       "ssl": {
         "title": "SSL Connection",
-        "description": "Encrypt data using SSL.",
+        "description": "Encrypt data using SSL. When activating SSL, please select one of the connection modes.",
         "type": "boolean",
         "default": false,
         "order": 6
       },
+      "ssl_mode": {
+        "title": "SSL modes",
+        "description": "SSL connection configuration modes. \n <b>disable</b> - don't care about security and don't want to pay the overhead for encryption\n <b>allow</b> - don't care about security but will pay the overhead for encryption if the server insists on it\n <b>prefer</b> - don't care about encryption but will pay the overhead of encryption if the server supports it\n <b>require</b> - want my data to be encrypted, and I accept the overhead. I trust that the network will make sure I always connect to the server I want\n  <b>verify-ca</b> - want my data encrypted, and I accept the overhead. I want to be sure that I connect to a server that I trust\n  <b>verify-full</b> - want my data encrypted, and I accept the overhead. I want to be sure that I connect to a server I trust, and that it's the one I specify\n See more information - <a href=\"https://jdbc.postgresql.org/documentation/head/ssl-client.html\"> in the docs</a>.",
+        "type": "object",
+        "order": 7,
+        "oneOf": [
+          {
+            "title": "Disable",
```

Comment:
nit: make the titles lowercase, so that they match the `sslmode` values and the description, which mentions them in lower case.

Reply:
thanks, I changed the titles of all modes in the specifications of the postgres connectors